platform-sdk/swirlds-merkledb/src/main/java/com/swirlds/merkledb/files/DataFileWriter.java

```diff
@@ -71,15 +85,20 @@ public final class DataFileWriter {
      * @param dataFileDir the path to directory to create the data file in
      * @param index the index number for this file
      * @param creationTime the time stamp for the creation time for this file
+     * @param compactionLevel the compaction level for this file
+     * @param dataBufferSize the size of the memory mapped data buffer to use for writing data items
      */
     public DataFileWriter(
             final String filePrefix,
             final Path dataFileDir,
             final int index,
             final Instant creationTime,
-            final int compactionLevel)
+            final int compactionLevel,
+            final long dataBufferSize)
             throws IOException {
-        this.path = createDataFilePath(filePrefix, dataFileDir, index, creationTime, DataFileCommon.FILE_EXTENSION);
+        this.dataBufferSize = dataBufferSize;
+        path = createDataFilePath(filePrefix, dataFileDir, index, creationTime, DataFileCommon.FILE_EXTENSION);
```

Comment:
To minimize the size of the current PR, this change is not needed (remove "this." and add an empty line)